### DISCUSSION TITLE
Progress Bar?

### DISCUSSION THREAD

**Discussion by benjibee** [spatie/ray#857] [Ideas]
I find myself using Ray pretty often to keep track of longer-running tasks and outputting information about them as they run. It would be pretty darn useful if it were possible to display a message and then update a progress bar within that message something akin to:

```
Task 1 (Update n with x)
[===-----------------] (12% complete)
```

Perhaps others would also find this useful, just want to put it out there.

**Comment by freekmurze** [comment]
We don't have a progress bar just yet, but I share something else. Here's how you can update an item in Ray

```
$ray = ray('counting down!');

foreach(range(10, 1) as $number) {
    sleep(1);
    $ray->send($number);
}
```

**Reply by benjibee** [reply]
That's pretty much what I needed, so there you go! Thanks :)
